### ISSUE TITLE
Frame drops when using haptics

### ISSUE THREAD

**Issue by mohammad-malik** [8bitPit/Niagara-Issues#1895]
### Describe the bug

When scrolling through the alphabetic app drawer list with haptic feedback on, it starts dropping frames and lagging. 

### Steps to reproduce

1. Use a 90 Hz or above refresh rate device
2. Install version 1.6.1 or above (versions with haptic feedback)
3. Turn on haptic feedback and scroll through the list.
4. Turn off haptic feedback and feel the difference. 

### Device name

OnePlus 7T Pro 

### Android version

Android 11

### Screenshots / Screen recordings

I would add a screen recording but that don't go above 60 fps which doesnt show the problem.

**Comment by TFackler** [comment]
Hello there,

we will be looking into this and then get back to you :)

**Comment by Oddward** [comment]
There should be a way to show your fps count on-screen in developer options, try enabling that to confirm the frame rate drop and maybe record it

**Comment by Yrlish** [comment]
I have started to experience the exact same bug when I woke up today with a new version with haptic feedback as default.

Here is a recording where you can clearly see the difference. With haptic feedback enabled the render method is definitely lagging behind. 

https://youtube.com/shorts/EjBc4VWsucM

**Comment by Maxr1998** [comment]
Indeed, I can reproduce frame drops when the haptic feedback is enabled on the scrollbar.
I think we can provide a workaround which should reduce the stuttering a bit, but I believe there's an underlying issue in the framework that causes this to occur in the first place.

**Comment by Msouza91** [comment]
Tried to replicate that on my device, but haven't been able to reproduce, enabled the frame counter on developer options and didn't have any frame drop with both settings, tried recording the evidence, but the screen recorder locks frame rate to 60 FPS.


https://user-images.githubusercontent.com/19522556/185794705-65e812af-c292-4db9-80ee-906a52a81cc3.mp4

I'm on A12 - S22 Ultra

**Comment by Maxr1998** [comment]
It could be that it's fixed on Android 12. I also don't notice it on my Z fold4 which has Android 12. I could reproduce it on my Pixel 5 though which is still on Android 11.
